src/main/java/com/semmle/jira/addon/workflow/LgtmTransitionNotificationFunction.java

```diff
@@ -45,14 +48,20 @@
   @SuppressWarnings("rawtypes")
   public void execute(Map transientVars, Map args, PropertySet ps) throws WorkflowException {
 
-    String configKey = "webhook"; // TODO get config key from custom field
+    Transition transition = getTransitionParam(args);
+    MutableIssue issue = getIssue(transientVars);
+
+    CustomField customField =
+        ComponentAccessor.getCustomFieldManager()
+            .getCustomFieldObjectByName(Constants.CUSTOM_FIELD_NAME);
+
+    String configKey = (String) issue.getCustomFieldValue(customField);
+
```

Comment:
insert `if (configKey == null) configKey = "webhook";` for backwards compatibility.